src/panels/config/automation/condition/types/ha-automation-condition-state.ts

```diff
@@ -21,6 +21,23 @@ export class HaStateCondition extends LitElement implements ConditionElement {
 
   protected render() {
     const { entity_id, attribute, state } = this.condition;
+    let trgFor = this.condition.for;
+
+    if (
+      trgFor &&
```

Comment:
Let give this variable a logical name, like `for`

Reply:
Maybe not exactly *that*, but something better, yes.
`forTime`?

Reply:
I agree and the variable is now renamed